### PR TITLE
Update subscriptions.json

### PR DESCRIPTION
**Pull request by rabbott01**
Updating "Subscription user" to "Subscription viewer" per RHCLOUD-33136

### CODE REVIEW

File: configs/prod/roles/subscriptions.json [hunk 1]
```diff
@@ -15,11 +15,11 @@
       ]
     },
     {
-      "name": "Subscriptions user",
+      "name": "Subscriptions viewer",
```
Comment (petracihalova) [inline configs/prod/roles/subscriptions.json:18]: please let the "name" same and add the "display_name" field as we did here https://github.com/RedHatInsights/rbac-config/pull/492

**Review by petracihalova — CHANGES_REQUESTED** [review]

**Comment by petracihalova** [comment]
> Thanks @petracihalova - I added the "display_name" to Stage/Prod configs.

awesome, thank you for adding "display_name" please can you change back the "name" field?

File: configs/prod/roles/subscriptions.json [hunk 2]
```diff
@@ -16,10 +16,11 @@
     },
     {
       "name": "Subscriptions user",
+      "display_name": "Subscription viewer",
```
Comment (petracihalova) [inline configs/prod/roles/subscriptions.json:19]:
```suggestion
      "display_name": "Subscriptions viewer",
```

**Comment by lpichler** [comment]
@rabbott01 if you can squash commit, it would be awesome!

maybe you can increment version by 1 (no by 2)

**Pull request merged**